### DISCUSSION TITLE
Override or block webpages from using / as an accesskey?

### DISCUSSION THREAD

**Discussion by fredrikjonson** [tridactyl/tridactyl#3355] [Q&A]
Some webpages use '/' as an accesskey for their on site search boxes. As a tridactyl user that is never what I want.

Is it possible to configure tridactyl or firefox to override or disallow webpages from using '/'? So that / always maps to the quick find box that searches page contents.

**Comment by bovine3dom** [comment]
We already try to do this. We're not sure why it doesn't work on all sites - see https://github.com/tridactyl/tridactyl/issues/904

**Reply by fredrikjonson** [reply]
Thanks, good to know. I should have searched the issue list of course.
